src/test/java/ReaderWriterTest.java

```diff
@@ -1,91 +1,112 @@
 package test;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
 import IO.DbReader;
 import IO.DbWriter;
 import data.Database;
 import data.DatabaseQuery;
 import java.io.BufferedReader;
-import java.io.File;
 import java.io.IOException;
-import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.io.Reader;
 import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
-import java.util.List;
-import org.junit.Assert;
-import org.junit.Before;
+import java.util.ArrayList;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-/* Validate that a .txt file can be properly be stored written and read from database.  */
+/* Validate that a .txt file can be properly be stored written and read from database. */
 public final class ReaderWriterTest {
-  private static String inputFile;
-  private static String outputFile;
-  private static InputStream input;
-  private static OutputStream output;
-
-  private Database database = new Database("test");
-  private DbWriter writer = new DbWriter(database, "test", "writingIn");
-  private DbReader reader = new DbReader(database, "test", "writingIn");
-
-  @Before
-  public void setUp() {}
 
   @Test
-  public void validateDatabaseName() {
+  public void validateDatabaseName() throws IOException {
```

Comment:
In tests, you can just write `throws Exception`. No one is writing code to call the test methods, so you don't have to document precisely which exceptions can be thrown.